dualscreeninfo/android/src/main/kotlin/com/microsoft/reactnativedualscreen/dualscreen/Hinge.kt

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.reactnativedualscreen.dualscreen
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.Rect
+import android.view.View
+import com.microsoft.device.display.DisplayMask
+
+class Hinge(context: Context) : View(context) {
+    private val mDisplayMask: DisplayMask
+        get() {
+            return DisplayMask.fromResourcesRectApproximation(context)
+        }
+    private val rotation: Int = 0
```

Comment:
>rotation: Int = 0 [](start = 16, length = 17)

What happens when the device is rotated?  Should this be querying the device for the orientation instead of supplying 0?

Reply:
sorry - forgot to push an update for this